docs/marketplace/user/features/202108.0/marketplace-shipment-feature-overview.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Marketplace Shipment feature overview
+description: This document contains concept information for the Marketplace Shipment feature.
+template: concept-topic-template
+---
+
+The *Marketplace Shipment* allows splitting the [marketplace order](/docs/marketplace/user/features/{{ page.version }}/marketplace-order-management/marketplace-order-management-overview/marketplace-order-management-overview.html) into different shipments based on merchants who will process it.
+
+A *shipment* is a set of two or more products combined by the same delivery address.
+
+{% info_block infoBox "Info" %}
```

Comment:
Please delete this note (see my comment above). Add the info from scoss doc here if needed